helm/examples/local/values-localfile.yaml

```diff
@@ -5,16 +5,43 @@
 # The pvc names are dynamic based on the install name.
 # In this example the install name was "localfile"
 
+#The following is a generated values yaml that you can capture and feed to the hpcc helm chart when installing.
+#it will configure the hpcc storage to use the planes and pvcs installed by this chart.
+#e.g.:
+#helm install myhpcc hpcc/hpcc --set global.image.version=latest --values values-localfile.yaml
 
 storage:
-  dllStorage:
-    existingClaim: "dll-localfile-hpcc-localfile-pvc"
-    forcePermissions: true
+  planes:
+  - name: dali
+    pvc: dali-localfile-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/dalistorage"
+  - name: dll
+    pvc: dll-localfile-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/queries"
+  - name: sasha
+    pvc: sasha-localfile-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/sasha"
+  - name: data
+    pvc: data-localfile-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/hpcc-data"
+    labels:
+    - data
```

Comment:
now out of date - labels missing from other entries?

Reply:
I've now added labels